app/src/main/java/kr/co/connect/boostcamp/livewhere/model/BookmarkUser.kt

```diff
@@ -0,0 +1,6 @@
+package kr.co.connect.boostcamp.livewhere.model
+
+
+data class BookmarkUser(
+    var uuid: String? = null
```

Comment:
uuid가 반드시 필요하다면, 아래와 같이 해주는 것이 어떨까요?
```suggestion
    val uuid: String
```